modules/comment/logic/commentLogic.js

```diff
@@ -1,7 +1,6 @@
-const { Comment, Profile } = require('../../../models');
+const { Comment, Profile, Tip } = require('../../../models');
```

Comment:
It does break the type suggestion in a way. But the imported types have the same interface anyways. The return types (eg. from `find` are not typed anyways for now. This will be changed when we move to typescript.